src/CalculatorGeneralItem/index.js

```diff
@@ -23,7 +23,7 @@ function CalculatorGeneralItem(props) {
           className={styles.saveButton}
           onClick={props.onSaveButtonClicked}
           >
-          LAGRE
+          {props.children}
         </Button>
```

Comment:
You already have props.children for the contents of your expanded component. Now your 'children' will appear in two places, one of them being inside the button. I would change this prop to be specific for the text it should display as Johannes suggested. We have used a naming convention like ´props.saveButtonText´